resources/views/email/verified.blade.php

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet">
+    <style>
+        html, body {
+            background-color: #fff;
+            color: #636b6f;
+            font-family: 'Nunito', sans-serif;
+            font-weight: 200;
+            height: 100vh;
+            margin: 0;
+        }
+        .container{
+            text-align: center;
+            height: 500px;
+        }
+        h3{
+            line-height: 500px;
+        }
+        a{
+            background-color: #2382d2;
+            color: white;
+            padding: 10px;
+            outline: none;
+            border: none;
+            border-radius: 5px;
+            cursor: pointer;
+            text-decoration: none;
+        }
+    </style>
+    <title>Email Verification</title>
+</head>
+<body>
+    <div class="container">
+        <h3>{{ $msg }}</h3>
+        <a href="{{env('FRONTEND_URL')}}" class="back">Back to home</a>
+        @if($allowResend)
+        <a href="{{ route('verification.resend', request()->id) }}">Resend</a>
+        @endif
+    </div>
+</body>
+</html>
```

Comment:
This file is unnecessary and should be removed.

Reply:
It's the file that render the view came from the email verification link

Reply:
I realize that, that's why I am saying it should be moved to the frontend.